src/Context/FeatureContext.php

```diff
@@ -67,6 +67,11 @@ class FeatureContext implements SnippetAcceptingContext {
 		'dbhost' => '127.0.0.1',
 	];
 
+	/**
+	 *  What type of database should WordPress use for the test installations. Default to MySQL
+	 */
+	private static $db_type = 'MYSQL';
```

Comment:
Probably personal preference, but I'd suggest lowercase `mysql` and `sqlite` values, so that one can do `DB_TYPE=sqlite ...` instead.

Reply:
@mrsdizzie Is there some advantage to setting this as a class-level variable instead of reading it via `getenv()` each time we need it? I think the codebase generally uses the latter, so it might be nice to maintain consistency...

Reply:
I see that we have `self::$db_settings`, so maybe my comment is invalid.

Reply:
I also think it is clearer this way because with getenv the logic is "is it sqlite or is it not sqlite" because there is no existing use of DB_TYPE=mysql anywhere. This way it can be "is it sqlite, is it mysql" and maybe something else someday in the future.